client/src/components/Common/Footer/style.css

```diff
@@ -0,0 +1,14 @@
+.footer-component{
+  background-color:transparent;
+  border-top: 1px solid #E7E7E7;
+  text-align: center;
```

Comment:
there is no need to this line because you already overwrite it in flex align item.